tests/test_functions.py

```
@@ -0,0 +1,137 @@
+        return "null"
+    elif type == Type.BOOL:
+        return choice(["true", "false"])
+    elif type == Type.INT:
+        return str(choice(range(-1000, 1000)))
+    elif type == Type.FLOAT:
+        return str(choice(list(map(lambda x: x / 10, range(-1000, 1000)))))
+    elif type == Type.STRING:
+        return f"\"{''.join(choice(string.ascii_letters + string.digits) for _ in range(choice(range(0, 100))))}\""
+    elif type == Type.LIST:
+        return str([random_value(choice(all_types)) for _ in range(choice(range(0, 10)))])
+    elif isinstance(type, Optional):
```

Pylint (3.3.7)

[warning] Second argument of isinstance is not a type

(W1116)